### PR TITLE
Libcanberra

### PR DESCRIPTION
**Pull request by scopatz**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by scopatz** [comment]
This is R4R @conda-forge/staged-recipes?

File: recipes/libcanberra/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  build:
+    - {{ compiler('c') }}
+    - make
+    - pkg-config
+  host:
+    - libvorbis
+    - libtool
+  run:
+    - libvorbis
+    - libtool
```
Comment (isuruf) [inline recipes/libcanberra/meta.yaml]: This needs a pinning

File: recipes/libcanberra/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - pkg-config
+  host:
+    - libvorbis
+    - libtool
+  run:
+    - libvorbis
```
Comment (isuruf) [inline recipes/libcanberra/meta.yaml]: Same here

**Comment by scopatz** [comment]
Pinnings added, ready for review again

File: recipes/libcanberra/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  build:
+    - {{ compiler('c') }}
+    - make
+    - pkg-config
+  host:
+    - libvorbis
+    - libtool
+  run:
+    - libvorbis >=1
+    - libtool >=2
```
Comment (isuruf) [inline recipes/libcanberra/meta.yaml]: This should probably be pinned to major version like,
`- {{ pin_compatible("libtool") }}`

File: recipes/libcanberra/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - pkg-config
+  host:
+    - libvorbis
+    - libtool
+  run:
+    - libvorbis >=1
```
Comment (isuruf) [inline recipes/libcanberra/meta.yaml]: Same here

**Comment by scopatz** [comment]
R4R again!

**Comment by isuruf** [comment]
I approved it and thought you'd merge after tests pass. :smile:

**Comment by scopatz** [comment]
haha will do so in future!

**Pull request merged**